qiskit_dynamics/models/generator_models.py

```diff
@@ -65,70 +61,52 @@ def frame(self) -> BaseFrame:
     @frame.setter
     @abstractmethod
     def frame(self, frame: BaseFrame):
-        """Set the frame."""
-        pass
-
-    @property
-    @abstractmethod
-    def cutoff_freq(self) -> float:
-        """Get cutoff frequency."""
+        """Set the frame; either an already instantiated :class:`Frame` object
+        a valid argument for the constructor of :class:`Frame`, or `None`.
+        Takes care of putting all operators into the basis in which the frame
+        matrix F is diagonal.
+        """
         pass
 
-    @cutoff_freq.setter
     @abstractmethod
-    def cutoff_freq(self, cutoff_freq: float):
-        """Set cutoff frequency."""
+    def evaluate_with_state(
```

Comment:
What are your thoughts on changing the names of these functions to:
- `evaluate_generator`, and
- `evaluate_rhs`

I'm just wary of the naming collision between the "state" of a differential equation (which is what this is referring to more generally) and a "quantum state". I think the above suggested names are very concrete/explicit (but am of course open to discussion).